nox.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2017 Joachim van der Herten
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import nox
+
+TEST_DEPS = ('six', 'parameterized', 'nbconvert', 'nbformat', 'jupyter', 'jupyter_client', 'matplotlib', 'nose')
```

Comment:
k, as discussed we could split matplotlib from this as it is only needed for the system tests